goss/goss-windows.yaml

```
@@ -38,6 +38,9 @@ command:
     exit-status: 0
     stdout:
       - 3.12.2
+  ruby:
+    exec: ruby -v
+    exit-status: 0
```

```suggestion
    exit-status: 0
   stdout:
     - 2.6.10
```
@dduportal Should it be restored too?